repos/actors/checkuptime/actor.py

```diff
@@ -0,0 +1,19 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import checkuptime
+from leapp.models import Uptime
+from leapp.reporting import Report
+from leapp.tags import IPUWorkflowTag, TargetTransactionChecksPhaseTag
+
+
+class CheckUptime(Actor):
+    """
+    No documentation has been provided for the uptime_scanner actor.
+    """
+
+    name = 'check_uptime'
+    consumes = ()
+    produces = (Report,)
+    tags = (TargetTransactionChecksPhaseTag, IPUWorkflowTag)
```

Comment:
Executing the actor in wrong phase. what this actor has common with the upgrade (dnf) transaction? from that, I would expect 2 actors - scanner and checker in first two phases